lib/taurus/qt/qtgui/button/test/res/Timeout

```diff
@@ -40,7 +40,7 @@ The argument value is the time that will take to execute the command.
 
 """
 
-import PyTango
+import PyTango  # Tango-centric
```

Comment:
Just an idea for the future: move this to tango scheme directory e.g. taurus.core.tango.test.res

Reply:
That is against sep6. 
IMHO, the proper solution requires of TEP13 (plugins support) being implemented, and then just make this test part of the tango plugin (so that if it is present is because tango is installed)

Reply:
You mean SEP5:
> Test directories may have a subdirectory named res, containing any necessary resource files for our test. A resource file could be for example: myresource.txt. There are no restrictions at all for the resource file names.

> res directories must contain an __init__.py file in order to ease resource handling.

You mean that "our test" should be interpreted as the test located in the test directory which is the parent directory of this res sub-directory. 
If I  remember well, when we were defining the SEP5 we were thinking of reusing resources between tests, even those located in different test directories. I think that for that reason we have added the __init__ module to the res directory so they could be easily located.